### PR TITLE
Update application friendly names in constants

### PR DESCRIPTION
**Pull request by martomo**
Revisited existing application entries and added several other mainstream/popular applications.

Main "breaking change" introduced in this pull request is that app constants previously with `_2` suffix have been changed to `_FIRETV` suffix, in case the application identifier is specific to Amazon devices.

### CODE REVIEW

File: androidtv/constants.py [hunk 1]
```diff
@@ -310,88 +316,128 @@
...
 APP_YOUTUBE = 'com.google.android.youtube.tv'
-APP_YOUTUBE_2 = 'com.amazon.firetv.youtube'
+APP_YOUTUBE_FIRETV = 'com.amazon.firetv.youtube'
+APP_YOUTUBE_KIDS = 'com.google.android.youtube.tvkids'
+APP_YOUTUBE_KIDS_FIRETV = 'com.amazon.firetv.youtube.kids'
+APP_YOUTUBE_MUSIC = 'com.google.android.youtube.tvmusic'
 APP_YOUTUBE_TV = 'com.google.android.youtube.tvunplugged'
 APP_ZIGGO_GO_TV = 'com.ziggo.tv'
 APPS = {APP_AE_TV: 'A&E',
+        APP_AMAZON_PRIME_VIDEO: 'Amazon Prime Video',
```
Comment (martomo) [inline androidtv/constants.py:371]: Renamed `APP_AMZ_VIDEO` as per discussion https://github.com/JeffLIrion/python-androidtv/pull/199#issuecomment-706712193.

Simply used the name associated to the application identifier on Google Play ([com.amazon.avod.thirdpartyclient](https://play.google.com/store/apps/details?id=com.amazon.avod.thirdpartyclient)).

File: androidtv/constants.py [hunk 2]
```diff
@@ -310,88 +316,128 @@
...
 APP_STEAM_LINK = 'com.valvesoftware.steamlink'
 APP_SYFY = 'com.amazon.webapps.nbc.syfy'
+APP_T2 = 'tv.perception.clients.tv.android'
+APP_TED = 'com.ted.android.tv'
+APP_TUNEIN = 'tunein.player'
 APP_TVHEADEND = 'de.cyberdream.dreamepg.tvh.tv.player'
-APP_TWITCH = 'tv.twitch.android.viewer'
-APP_TWITCH_APP = 'tv.twitch.android.app'
+APP_TWITCH = 'tv.twitch.android.app'
+APP_TWITCH_FIRETV = 'tv.twitch.android.viewer'
```
Comment (martomo) [inline androidtv/constants.py:354]: It seems that **tv.twitch.android.viewer** is the identifier for an old version (last update seems to be in 2015) of Twitch which apparently still comes installed with Amazon devices.

**Review by JeffLIrion — APPROVED** [review]
Looks good.  Any more changes or should I merge it now?

**Comment by martomo** [comment]
That's it for now, feel free to merge it at your own convenience.

**Pull request merged**